plone/app/iterate/configure.zcml

```diff
@@ -46,20 +46,6 @@
       directory="profiles/default"
       />
 
-  <!--
-    Remove, when CMFPlone no more depends on this profile.
-    Products.CMFPlone/Products/CMFPlone/MigrationTool.py:99:
-    Addon(profile_id=u'plone.app.iterate:plone.app.iterate')
-  -->
-  <genericsetup:registerProfile
-      name="plone.app.iterate"
```

Comment:
Please keep this profile and the related for now.  We have only implemented the real cleanup in a `plone.app.upgrade` step in Plone 6.0.3.
See https://github.com/plone/plone.app.iterate/issues/99#issuecomment-1484686642

Reply:
Oh, sorry, I thought, although I did not search for when that happened (🤚🏾 lazy me 😓 ).

After having seen a few >2015 old changes that were still lying around I assumed it was yet another one, sorry, will revert 😄 

Though that will bring unfortunately CMFPlone dependency in 😕